pkgs/development/python-modules/mip/cffi-version.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/pyproject.toml b/pyproject.toml
+index 489831f..13ae3a5 100644
+--- a/pyproject.toml
++++ b/pyproject.toml
+@@ -32,7 +32,7 @@ classifiers = [
+ ]
+ dynamic = ["version"]
+ 
+-dependencies = ["cffi==1.15.0"]
++dependencies = ["cffi==1.15.*"]
```

Comment:
Could that be only a `substituteInPlace` rather than a proper patch?

Reply:
Most definitely. I was biased towards patches because of a previous pull request, but I guess patching a single line is... simply too much.

Will convert to a scripted substitution.